Makefile

```diff
@@ -77,11 +77,8 @@ generate:
 	$(MAKE) gen-manifests
 	$(MAKE) fix-changed
 
-gen-manifests: bin/helm
```

Comment:
Doesn't changing this prevent me from running `make OPERATOR_VERSION=foo gen-manifests`?

Reply:
reverting those changes as they are not essential to this PR